src/test/java/com/g4s8/ghman/data/DatabaseExtension.java

```diff
@@ -41,23 +43,41 @@
     private final PGSimpleDataSource src = new PGSimpleDataSource();
 
     /**
-     * Embedded postgres.
+     * Testcontainers postgres.
      */
-    private final EmbeddedPostgres postgres = new EmbeddedPostgres(Version.V11_1);
+    private final PostgreSQLContainer postgres = new PostgreSQLContainer<>()
+        .withUsername("test_user")
+        .withPassword("test_pass");
 
     @Override
     public void beforeEach(final ExtensionContext context) throws Exception {
-        this.src.setUrl(this.postgres.start());
-        Flyway.configure().dataSource(this.src).load().migrate();
+        this.postgres.start();
+        this.src.setUrl(
+            new Joined(
+                "",
+                this.postgres.getJdbcUrl(),
+                "?user=",
+                this.postgres.getUsername(),
+                "&password=",
+                this.postgres.getPassword()
+            ).asString()
+        );
```

Comment:
@scristalli btw will not it work if you set user and password by calling `setUser` and `setPassword` on `src` to avoid this joining? May you even will not need to specify username and password in `afterEach` when getting connection?

Reply:
@olenagerasimova you're right, I was so busy [discussing the issue](https://github.com/g4s8/ghman/issues/52) of not having these methods on plain `DataSource` objects that I completely missed them in `PGSimpleDataSource`. Addressed in commit 60028f9.